.github/workflows/ci-circt-nightly.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         id: determine-branch
         shell: bash
         run: |
-          branch=$(git rev-parse --abbrev-ref HEAD)
+          branch=$(git rev-parse HEAD)
```

Comment:
This is necessary because other jobs will use this value as git `ref` to checkout, and will fail if it's a branch name from a forked repo. Replacing with a commit ID to avoid the failure.